### ISSUE TITLE
Citations format not spacing in-text citations properly, no spaces between numbers.

### ISSUE THREAD

**Issue by mjsielerjr** [jgm/pandoc-citeproc#445]
Hi there,

Just downloaded pandoc-citeproc and still getting the hang of everything.
I'm using MacOS, Atom text editor (with Zotero-citations), nature.csl, Zotero (with better bibtex) for my exporting my citations from markdown to docx files. When I export my document the in text citations aren't being spaced properly (see below). 

**How I'm citing in the document:**
<img width="261" alt="image" src="https://user-images.githubusercontent.com/6273555/79922908-541c9780-83e9-11ea-8b44-f4a14d584171.png">

**Command line arguments:**
`pandoc filename.md -f markdown -t docx -s -o filename.docx --bibliography library.bib`

**Expected behavior:**

example sentence<sup>1, 2, 3</sup>.

**Actual behavior:**

example sentence<sup>1,2,3</sup>.

**Screenshot of problem:**
![image](https://user-images.githubusercontent.com/6273555/79922877-3d764080-83e9-11ea-893b-9c96b9cfa9c3.png)

Thank you. Please let me know if I need to include additional information.

**Comment by jgm** [comment]
properly according to what standard?
And what CSL are you using?  This isn't the output you'd get with the default pandoc CSL, which is an author-date style.  This looks like  footnote style or maybe a numerical style?

**Comment by njbart** [comment]
`nature.csl`, apparently. Not sure where the OP’s expectations come from, but the journal Nature itself definitely does not use spaces after commas separating citation numbers. See, e.g., this article, published 15 April 2020: https://www.nature.com/articles/d41586-020-01000-9; check out both html and pdf versions.

**Comment by nathanlesage** [comment]
> nature.csl, apparently.

This style by default uses `collapse="citation-number"` to make sure to convert `1,2,3` to `1–3`, so I'm assuming it's not the most recent version from the Zotero style repo.

Nevertheless, if you change (currently) line 104 from `<citation collapse="citation-number">` to `<citation>`, there will be commata in the form of `1,2,3`. If you want spacing, you can check out (currently) line 108-110 and change the delimiter to include a space:

```xml
    <layout vertical-align="sup" delimiter=", ">
      <text variable="citation-number"/>
    </layout>
```

Will render `1, 2, 3`.

***

Long story short: Nothing wrong with Citeproc here, it depends upon the CSL style being used. Most of the way formatting works is determined by the CSL, and Citeproc in all its variations simply adhere to the rules defined in those CSL files!

**Comment by mjsielerjr** [comment]
Thanks for your help. Like I said, I'm still new to pandoc and getting the hang of things. The paper I was looking at had spaces between the superscripted in-text citations, but apparently this Scientific Reports (published by Nature) does things slightly differently. That's my mistake. Thanks for the tips about changing it though if I need to. Cheers.
